setup/updating.md

```diff
@@ -37,12 +37,20 @@ Updating Coder is a two-step process:
     ```bash
     helm repo update
     ```
+    
+2. Export the current helm chart values into a file:
 
-2. Upgrade to the desired version (i.e., `1.16.0`):
+    ```bash
```

Comment:
I think we should use `console` instead of `bash` as these are commands that someone is typing in manually

https://github.com/github/linguist/blob/8624ea022d3c78a5e8687efede4ea64a8b5cae1e/lib/linguist/languages.yml#L5314-L5325

Here's what this looks like with bash:

```bash
helm get values --namespace coder coder > current-values.yml
```

And the same with console:

```console
$ helm get values --namespace coder coder > current-values.yml
```

(I like to use `$` to show that this can be run as a non-root user)

All that said, I understand we might not want to change this now, because we would want things to be consistent across the docs... so leaving as `bash` is fine for now

Reply:
@khorne3 this is a good note for the future as well. We could go through and change all of them if we wanted, but that's a bit outside the scope of this PR. Perhaps moving forward we can update them all? Let me know your thoughts.

Reply:
@khorne3 I like this idea. I created a [CH card](https://app.clubhouse.io/coder/story/8969/change-code-fencing-from-bash-to-console) so this can be tracked (definitely think it should all be done at once)